x-pack/test/functional/apps/discover/value_suggestions.ts

```diff
@@ -27,16 +27,20 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('value suggestions', function describeIndexTests() {
     before(async function () {
+      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.load('x-pack/test/functional/es_archives/dashboard/drilldowns');
+
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/dashboard_drilldowns/drilldowns'
+      );
       await kibanaServer.uiSettings.update({
         'doc_table:legacy': true,
       });
     });
 
     after(async () => {
-      await esArchiver.unload('x-pack/test/functional/es_archives/dashboard/drilldowns');
       await kibanaServer.uiSettings.unset('doc_table:legacy');
```

Comment:
wouldn't the cleanStandardList fuction also reset advanced settings? Or just clears other saved objects only?

Reply:
@marius-dr good question. This is the list - so other saved objects. standard_list="url,index-pattern,action,query,alert,graph-workspace,tag,visualization,canvas-element,canvas-workpad,dashboard,search,lens,map,cases,uptime-dynamic-settings,osquery-saved-query,osquery-pack,infrastructure-ui-source,metrics-explorer-view,inventory-view,infrastructure-monitoring-log-view,apm-indices"